### PR TITLE
Navigation & datahandling

### PR DESCRIPTION
**Pull request by dride-for-sure**
Related to #50 & #78

(Well ... someday i will succeed on „a branch from a branch“ ... merge happens way toooo late :( )

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,56 +1,57 @@
...
-          <ProtectedRoute path="/cms/schools/search/:search" exact>
-            <SchoolSearchResults />
-          </ProtectedRoute>
-          <ProtectedRoute path="/cms/schools/:number" exact>
-            <SchoolDetails />
-          </ProtectedRoute>
-          <ProtectedRoute path="/cms/schools" exact>
-            <SchoolList />
-          </ProtectedRoute>
+    <>
```
Comment (christophersiem) [inline frontend/src/App.js:14]: fragments not needed here

```suggestion
 
```

File: frontend/src/components/lists/cms/school/SchoolList.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      <List>
+        {schools.map((school) => <SchoolListItem key={school.number} school={school} />)}
+      </List>
+    </Container>
+  );
+}
+
+const Container = styled.div``;
+
+const List = styled.ol``;
```
Comment (christophersiem) [inline frontend/src/components/lists/cms/school/SchoolList.js:17]: not (yet?) needed
```suggestion
```

File: frontend/src/containers/cms/Schools.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import CmsGrid from '../../components/structure/cms/Grid';
+import CmsWrapper from '../../components/structure/cms/Wrapper';
+import MaxWidth from '../../components/structure/MaxWidth';
+import { getSchoolByNumber, getSchoolByType, listSchools, listTypes } from '../../services/private/schoolApiService';
+
+export default function SchoolsOverview() {
+  const [types, setTypes] = useState();
+  const [schools, setSchools] = useState();
+  const [school, setSchool] = useState();
+  const { type, number } = useParams();
```
Comment (christophersiem) [inline frontend/src/containers/cms/Schools.js:16]: Consider setting an inital state explicitly. IMO that improves code readability

**Review by christophersiem — APPROVED** [review]
nice 🕺🏻 found some unused styled elements, but I think you already created them to style later?

**Comment by dride-for-sure** [comment]
Thanks @christophersiem!

**Pull request merged**